### PR TITLE
Alha dev

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

File: src/com/hms_networks/americas/sc/taginfo/TagInfo.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      this.groupToUse = TagGroup.C;
+    } else if (isInGroupD) {
+      this.groupToUse = TagGroup.D;
+    } else {
+      this.groupToUse = TagGroup.NONE;
+    }
+
+    if ((isInGroupA ? 1 : 0) + (isInGroupB ? 1 : 0) + (isInGroupC ? 1 : 0) + (isInGroupD ? 1 : 0)
+        > 1) {
+      System.out.println("Tag " + name + " is configured for multiple tag groups.");
```
[the remark this answers is not in the document]
Comment (TomKimsey) [reply src/com/hms_networks/americas/sc/taginfo/TagInfo.java:76]: I dont like the idea of libraries just printing stuff

[the remark this answers is not in the document]
Comment (alexjhawk) [reply src/com/hms_networks/americas/sc/taginfo/TagInfo.java:76]: Id say maybe have a setLogger method and if thats called, use the logger, else just do standard system.out.

[the remark this answers is not in the document]
Comment (alexjhawk) [reply src/com/hms_networks/americas/sc/taginfo/TagInfo.java:76]: I think if anything here, it'd be better to just eliminate this message all together, and have support for multiple tag groups, which is more in line with native flexy functionality actually anyways

[the remark this answers is not in the document]
Comment (alexjhawk) [reply src/com/hms_networks/americas/sc/taginfo/TagInfo.java]: support for multiple tag groups has been added. Message was removed

**Pull request merged**